test/e2e_node/restart_test.go

```diff
@@ -265,5 +269,114 @@ var _ = SIGDescribe("Restart [Serial] [Slow] [Disruptive]", func() {
 				}
 			}
 		})
+		// Regression test for https://issues.k8s.io/116925
```

Comment:
I think the test should eventually be marked Disruptive and Serial as it restarts Kubelet.

Reply:
This is already done, all the restart tests are marked as serial and disruptive - https://github.com/kubernetes/kubernetes/blob/2f6c4f5/test/e2e_node/restart_test.go#L70